addons/l10n_be/data/template/account.tax-be.csv

```diff
@@ -373,3 +373,11 @@
 "","","","","","","","","","","","tax","invoice","","","","",""
 "","","","","","","","","","","","base","refund","-83||-87||+85","","","",""
 "","","","","","","","","","","","tax","refund","","","","",""
+"attn_VAT-IN-V83-277-ROW-CC","2310","21% A EU Car (35% deductible)","21%","21% EU Car","21.0","percent","purchase","tax_group_tva_21","invest","False","base","invoice","+82||+88","","","",""
+"","","","","","","","","","","","tax","invoice","+59","a411","35","",""
+"","","","","","","","","","","","tax","invoice","-55","a451055","-100","",""
+"","","","","","","","","","","","tax","invoice","+82","","65","",""
+"","","","","","","","","","","","base","refund","-82||+84||-88","","","",""
+"","","","","","","","","","","","tax","refund","-62","a411","35","",""
+"","","","","","","","","","","","tax","refund","-62","a451055","-100","",""
+"","","","","","","","","","","","tax","refund","-82","","65","",""
```

Comment:
now it's missing the end of file line :joy: You can set up you IDE to add that for you, it's a coding convention we have every file should end with ONE empty line :smile:

Reply:
I have solved the problem and added a new line